### ISSUE TITLE
Reviewing Parsed DOM for contrast violations

### ISSUE THREAD

**Issue by pandasa123** [priyajedi/web-doctor#6]
After DOM manipulation and traversal is possible, as well as implementing WCAG contrast into python, we can review contrast violations for a web page
